pennylane/transforms/qcut.py

```diff
@@ -939,6 +940,142 @@ def qcut_processing_fn_mc(
     return qml.math.convert_like(np.mean(expvals), res0)
 
 
+@batch_transform
+def cut_circuit_mc(
+    tape: QuantumTape,
+    shots: int,
+    device_wires: Wires,
+    max_depth: int = 1,
+    classical_processing_fn: callable = None,
+) -> Tuple[Tuple[QuantumTape], Callable]:
+    """
+    Cut up a circuit containing sample measurements into smaller fragments.
+
+    Following the approach of `Peng et al. <https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.125.150504>`__,
+    strategic placement of :class:`~.WireCut` operations can allow a quantum circuit to be split
+    into disconnected circuit fragments. This transform allows for sampled measurement outcomes
+    to be recombined to full bitstrings and if a classical processing function is supplied,
+    an expectation value will be evaluated.
+
+    Args:
+        tape (QuantumTape): the tape of the full circuit to be cut
+        shots (int): number of shots
+        device_wires (Wires):  wires of the device that the cut circuits are to be run on
+        max_depth (int): the maximum depth used to expand the circuit while searching for wire cuts
+        classical_processing_fn (callable): a classical postprocessing function to be applied to
+            the reconstructed bitstrings. The expected input is a bitstring; a flat array of length `wires`
+            and the output should be a single number within the interval [-1, 1]
+
+    Returns:
+        Tuple[Tuple[QuantumTape], Callable]: the tapes corresponding to the circuit fragments as a
+        result of cutting and a post-processing function which either recombines samples to a full
```

Comment:
```suggestion
        result of cutting and then random expansion at :class:`~.MeasureNode` and :class:`~.PrepareNode` pairs,
        and a post-processing function which either recombines samples to a full
```
Feel free to reword a bit, just thought it'd be useful to mention.